RevenueCatUI/Templates/V2/ViewHelpers/Fill.swift

```diff
@@ -14,7 +14,7 @@
 import RevenueCat
 import SwiftUI
 
-#if PAYWALL_COMPONENTS
+#if !os(macOS) && !os(tvOS) // For Paywalls V2
```

Comment:
I  am commenting here because this is where GitHub let's me.
I updated on my other PR line 33 `self.fill()` because it wasn't compiling either. MAYBE we can see if it works now without it?

@MarkVillacampa